samples/sensor/hx711/README.rst

```diff
@@ -0,0 +1,99 @@
+.. _hx711:
+
+HX711: 24-Bit Analog-to-Digital Converter (ADC) for Weigh Scales
+################################################################
+
+Overview
+********
+This sample makes use of an HX711.
+Offset is calculated without any weight put on the scale.
+Then the user is prompted to put the CONFIG_HX711_CALIBRATION_WEIGHT on the scale and
+calibration is performed.
+
+After calibration :
+- Sampling rate is set to 10Hz
+- A measurement is performed
+
+- Sampling rate is set to 80Hz
+- A measurement is performed
+
+- Sampling rate is set to 10Hz
+- HX711 is power cycled
+- A measurement is performed
+
+- Sampling rate is set to 80Hz
+- HX711 is power cycled
+- A measurement is performed
+
+- Sampling rate is set to 10Hz
+- HX711 is powered OFF
+- A measurement is performed which returns -EACCES (-13)
+- Sampling rate is set to 80Hz
+- A measurement is performed which returns -EACCES (-13)
+
+- sleep 10s and repeat
+
+Requirements
+************
+
+This sample uses the HX711 sensor module from AliExpress, although any HX711 hardware implementation should work.
```

Comment:
"HX711 sensor module from AliExpress" Please provide more detailed information (maker reference would be better than the shop adress)

Reply:
I agree about not posting an URL for the AliExpress shop, but my module does not really have a model number on it..

Should I just remove the URL?

Reply:
Added board model used : "shengyang XFW-HX711"